src/app/templates/app/submit_new_license.html

```diff
@@ -224,8 +224,15 @@
   });
 </script>
 <script type="text/javascript">
+function removeDiv(elem){
+    $(elem).parent('div').remove();
+} 
+$(document).on('click', '#removeSourceUrl', function(){
+removeDiv(this);
+});
+
 $(document).on('click', '#addSourceUrl', function(){
-    var html = '<div class="col-sm-12 btn-group" style="padding:6px 0; margin-top:2"><input type="text" class="form-control" id="sourceUrl" placeholder="" name = "sourceurl"></div>';
+    var html = '<div class="col-sm-12 btn-group" style="padding:6px 0; margin-top:2"><input type="text" class="form-control" id="sourceUrl" placeholder="" name = "sourceurl"><span class="help-block" id="removeSourceUrl" style="color:red; text-align: left; font-weight: bold; font-style: italic; cursor:pointer;" >- remove</span> </div>';
```

Comment:
Can we instead show a "cross" sign (x) next to the row instead of "- remove" in red.

Reply:
@rtgdk I have made the changes.